pylsp/plugins/rope_autoimport.py

```diff
@@ -206,6 +241,80 @@ def _sort_import(score: int) -> str:
     return "[z" + str(score).rjust(_score_pow, "0")
 
 
+def get_name_or_module(document, diagnostic) -> str:
+    return (
+        parso.parse(document.lines[diagnostic["range"]["start"]["line"]])
+        .get_leaf_for_position((1, diagnostic["range"]["start"]["character"] + 1))
+        .value
+    )
```

Comment:
```suggestion
    start = diagnostic["range"]["start"]
    return (
        parso.parse(document.lines[start["line"]])
        .get_leaf_for_position((1, start["character"] + 1))
        .value
    )
```

Small refactorint to make this code more readable.